main.py

```diff
@@ -1,54 +1,129 @@
-from pypanther.wrap import exclude, include
+from pypanther.rules.aws_cloudtrail_rules.aws_console_root_login import AWSConsoleRootLogin
+
+from rules.aws_rules.alb_high_400s import AWSALBHighVol400s
+from rules.custom_logs.ids_rules import HostIDSBaseRule, HostIDSMalware
+from helpers.cloud import prod_account_ids, account_lookup_by_id, update_account_id_tests
 
-from rules.aws_cloudtrail_rules.my_first_custom_rule import MyFirstCustomRule
-from rules.aws_cloudtrail_rules.my_first_tuned_rule import MyFirstTunedRule
 
 ########################################################
-## Variables
+## Importing Panther-managed Rules
+##
+## Use get_panther_rules() to return a list of Panther-managed rules.
+## Calling without arguments returns everything and passing attributes
+## like LogTypes, Severity, or Tags filters rules matching the values.
 
 onboarded_log_types = [
-    # replace with your log types
+    # Replace with your onboarded log types
     PantherLogType.AWS_CloudTrail,
+    PantherLogType.AWS_GuardDuty,
+    PantherLogType.Okta_SystemLog,
 ]
 
+# Get Panther built-in rules for onboarded log types
+base_rules = get_panther_rules(LogTypes=onboarded_log_types)
+
+## Get Panther-managed rules with certain severities
+# high_sev_rules = get_panther_rules(
+#     LogTypes=onboarded_log_types,
+#     Severity=[
+#         PantherSeverity.Critical,
+#         PantherSeverity.High,
+#     ],
+# )
+
+## Get all Panther-managed Rules
+# all_rules = get_panther_rules()
+
+
+########################################################
+## Filter/Dynamic Functions
+##
+## Add filters to include/exclude events from being evaluated by rules.
+
+
+def prod_account(event):
+    # TODO() Change this to use event.udm('account_id')
+    return event.get("recipientAccountId") in prod_account_ids
+
+
+sensitive_services = {"s3", "dynamodb", "iam", "secretsmanager", "ec2"}
+
+
+def guard_duty_sensitive_service(event):
+    service_name = event.deep_get("service", "action", "awsApiCallAction", "serviceName")
+    return any(service_name.startswith(service) for service in sensitive_services)
+
+
+def root_login_account(_, event):
+    ip_address = event.get("sourceIPAddress")
+    account = account_lookup_by_id(event.get("recipientAccountId"))
+    return f"Root Login from [{ip_address}] in account [{account}]"
+
+
 ########################################################
 ## Overrides
+##
+## Apply your custom configurations to Panther-managed rules.
+## This can include single or multiple attributes and filters.
 
-AWSCloudTrailAccountDiscovery.Severity = PantherSeverity.High
-AWSCloudTrailCodebuildProjectMadePublic.DedupPeriodMinutes = 10
+# Override a single attribute
+AWSCloudTrailAccountDiscovery.Severity = PantherSeverity.Low
 
+# Override a set of rule attributes and attach a filter
+include(prod_account)(AWSCloudTrailStopped)
+# TODO() Update this temporary workaround
+update_account_id_tests([AWSCloudTrailStopped])
```

Comment:
This seems fragile... The way `update_account_id_tests` is written it can only update tests with a sample prod account.  If you wanted to `include(dev_account)` for example it wouldn't work.

Reply:
Yeah, it is fragile. Max has an RFC out on how to make it better! This is just temporary (as the comment suggests)